app/src/androidTest/java/com/github/h3lp3rs/h3lp/GuideTest.kt

```diff
@@ -1,54 +1,49 @@
 package com.github.h3lp3rs.h3lp
 
-import android.content.Context
-import androidx.preference.PreferenceManager
-import androidx.test.core.app.ApplicationProvider
-import androidx.test.espresso.Espresso.onView
-import androidx.test.espresso.assertion.ViewAssertions
-import androidx.test.espresso.assertion.ViewAssertions.matches
-import androidx.test.espresso.matcher.ViewMatchers
-import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
-import androidx.test.espresso.matcher.ViewMatchers.withText
-import androidx.test.ext.junit.rules.ActivityScenarioRule
+import android.content.Intent
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ActivityScenario.*
+import androidx.test.core.app.ApplicationProvider.*
 import androidx.test.ext.junit.runners.AndroidJUnit4
-import androidx.test.platform.app.InstrumentationRegistry
-import androidx.test.uiautomator.UiDevice
-import androidx.test.uiautomator.UiSelector
-import junit.framework.TestCase.assertFalse
+import com.github.h3lp3rs.h3lp.database.Databases.*
+import com.github.h3lp3rs.h3lp.database.MockDatabase
+import com.github.h3lp3rs.h3lp.signin.SignInActivity.Companion.globalContext
+import com.github.h3lp3rs.h3lp.signin.SignInActivity.Companion.userUid
+import com.github.h3lp3rs.h3lp.storage.Storages.*
+import com.github.h3lp3rs.h3lp.storage.Storages.Companion.resetStorage
+import com.github.h3lp3rs.h3lp.storage.Storages.Companion.storageOf
 import junit.framework.TestCase.assertTrue
-import org.junit.Rule
+import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
 class GuideTest {
-    @get:Rule
-    val testRule = ActivityScenarioRule(
-        MainPageActivity::class.java
-    )
 
-    private val targetContext : Context = ApplicationProvider.getApplicationContext()
-
-    private fun clearPreferences() {
-        val sharedPrefs = PreferenceManager.getDefaultSharedPreferences(targetContext)
-        val editor = sharedPrefs.edit()
-        editor.clear()
-        editor.apply()
+    private fun launch(): ActivityScenario<MainPageActivity> {
+        return launch(Intent(getApplicationContext(), MainPageActivity::class.java))
```

Comment:
Could possibly be refactored out since it's sued in several test files

Reply:
Same as my other replies